day_two_guide/topics/scaling_etcd.adoc

```diff
@@ -0,0 +1,432 @@
+member 2a529ba1840722c0 is healthy: got healthy result from https://192.168.55.8:2379
+member ed4f0efd277d7599 is healthy: got healthy result from https://192.168.55.13:2379
+cluster is healthy
+----
++
+Or, using etcd v3 API:
++
+----
+# ETCDCTL_API=3 etcdctl --cert="/etc/etcd/peer.crt" \
+          --key=/etc/etcd/peer.key \
+          --cacert="/etc/etcd/ca.crt" \
+          --endpoints="https://*master-0.example.com*:2379,\
+            https://*master-1.example.com*:2379,\
+            https://*master-2.example.com*:2379"
+            endpoint health
+https://master-0.example.com:2379 is healthy: successfully committed proposal: took = 5.011358ms
+https://master-1.example.com:2379 is healthy: successfully committed proposal: took = 1.305173ms
+https://master-2.example.com:2379 is healthy: successfully committed proposal: took = 1.388772ms
+----
+
+. Before running the `scaleup` playbook, ensure the new host is registered to
+the proper Red Hat software channels:
++
+----
+# subscription-manager register \
+    --username=*<username>* --password=*<password>*
+# subscription-manager attach --pool=*<poolid>*
+# subscription-manager repos --disable="*"
+# subscription-manager repos \
+    --enable=rhel-7-server-rpms \
+    --enable=rhel-7-server-extras-rpms
+----
++
+etcd is hosted in the `rhel-7-server-extras-rpms` software channel.
+
+==== Adding a new etcd host using Ansible
+
+[discrete]
+== Procedure
+
+. Modify the Ansible inventory file and create a new group named `[new_etcd]`
+and add the new host. Then, add the `new_etcd` group as a child of the `[OSEv3]`
+group:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+<new_etcd>
+
+... [OUTPUT ABBREVIATED] ...
+
+[etcd]
+master-0.example.com
+master-1.example.com
+master-2.example.com
+
+[new_etcd]
+etcd0.example.com
+----
+
+. Run the etcd `scaleup` playbook from the host that executed the initial
+installation and where the Ansible inventory file is:
++
+----
+$ ansible-playbook  /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-etcd/scaleup.yml
+----
+
+. Once the above has finished, modify the inventory file to reflect the current
+status by moving the new etcd host from the `[new_etcd]` group to the `[etcd]`
+group:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+<new_etcd>
+
+... [OUTPUT ABBREVIATED] ...
+
+[etcd]
+master-0.example.com
+master-1.example.com
+master-2.example.com
+etcd0.example.com
+----
+
+. If using Flannel, modify the `flanneld` service configuration, located at
+`/etc/sysconfig/flanneld` on every {product-title} host, to include the new etcd
+host:
++
+----
+FLANNEL_ETCD_ENDPOINTS=https://master-0.example.com:2379,https://master-1.example.com:2379,https://master-2.example.com:2379,https://etcd0.example.com:2379
+----
+
+. Restart the `flanneld` service:
++
+----
+# systemctl restart flanneld.service
+----
+
+==== Manually adding a new etcd host
+
+The following steps can be performed on any etcd member. If using the Ansible
```

Comment:
So, do we perform these steps on any etcd member or the first host in the Ansible inventory?
Which Ansible installer are you talking about?